streamalert/rules_engine/rules_engine.py

```diff
@@ -346,18 +349,22 @@ def _configure_publishers(cls, rule):
 
         Args:
             rule (Rule): The rule to create publishers for
+            requested_outputs (set): A set containing the outputs
 
         Returns:
             dict: Maps string outputs names to lists of strings of their fully qualified publishers
         """
-        requested_outputs = rule.outputs_set
         requested_publishers = rule.publishers
         if not requested_publishers:
             return None
 
         configured_publishers = {}
 
         for output in requested_outputs:
+            if output == "aws-firehose:alerts":
```

Comment:
I actually think you can omit this. That way no hard coded "skipped" outputs that we have to hunt down

Reply:
This only skips on the `publisher`, since this change, impacted the publishers there were thrown back. `aws-firehose:alerts` came back with `[]` which seemed pointless as it didn't need a `publisher`